src/core/mesh/qgsmeshlayertemporalproperties.cpp

```diff
@@ -81,6 +81,11 @@ QgsDateTimeRange QgsMeshLayerTemporalProperties::calculateTemporalExtent( QgsMap
   return mTimeExtent;
 }
 
+QgsTemporalProperty::Flags QgsMeshLayerTemporalProperties::flags() const
+{
+  return QgsTemporalProperty::Flags() | QgsTemporalProperty::FlagDontInvalidateCachedRendersWhenRangeChanges;
```

Comment:
FlagDontInvalidateCachedRendersWhenRangeChanges -> FlagDontInvalidateRendererCacheWhenRangeChanges

Reply:
This isn't the correct use of this flag -- this flag was designed with a single use case in mind, where an entire layer's visibility is controlled by the temporal framework but where the actual appearance of that layer doesn't change regardless of what the actual range is. I.e. it's used only for rasters and vectors set to display for a fixed time range only -- but avoid the cache invalidation we ensure that these layers don't need to be needlessly re-rendered when the time range changes, but instead just selectively use the cached image or not based on the layer visibility...